eventuate-util-swagger-ui-tests/src/test/java/io/eventuate/util/swaggerui/tests/SwaggerUiAvailabilityTest.java

```diff
@@ -27,12 +25,7 @@
 
   @Test
   public void testSwaggerUiAvailability() throws IOException {
-    assertUrlStatusIsOk(String.format("http://%s:%s/swagger-ui/index.html", "localhost", port));
+    UrlTesting.assertUrlStatusIsOk("http://localhost", port, "swagger-ui/index.html");
```

Comment:
I think this would be better as 

```
    UrlTesting.assertUrlStatusIsOk("localhost", port, "/swagger-ui/index.html")
```

* No need to always write `http://`.
* Paths begin with '/'

Reply:
Fixed